# This patch file was generated by NetBeans IDE
# Following Index: paths are relative to: \\192.168.162.128\cms\server\INFOBAECMSR\sites\all\modules\contrib\apc
# This patch can be applied using context Tools: Patch action on respective folder.
# It uses platform neutral UTF-8 encoding and \n newlines.
# Above lines and this line are ignored by the patching process.
Index: drupal_apc_cache.inc
--- drupal_apc_cache.inc Base (BASE)
+++ drupal_apc_cache.inc Locally Modified (Based On LOCAL)
@@ -254,7 +254,9 @@
   }
 
   function clear($cid = NULL, $wildcard = FALSE) {
-    if (drupal_is_cli() && function_exists('drush_log')) {
+    $warning_shown = &drupal_static('apc_warning_shown', FALSE);
+    if (drupal_is_cli() && function_exists('drush_log') && !$warning_shown) {
+      $warning_shown = TRUE;
       drush_log($this->bin . '(' . $cid . ') was not cleared. APC cli uses a different memory storage then the webserver. For more info see: http://drupal.org/node/1278232', 'warning');
       return;
     }
